.gitignore

```diff
@@ -52,3 +52,6 @@ pkg/
 # Ignore generated yard docs
 .yardoc/
 doc/
+
+# Ignore Gemfile.lock for gem development
+Gemfile.lock
```

Comment:
Why we removed this file for a gemfile: https://yehudakatz.com/2010/12/16/clarifying-the-roles-of-the-gemspec-and-gemfile/